packages/fhir-converter/src/templates/cda/Sections/Medication.hbs

```diff
@@ -30,7 +30,7 @@
                                         {{>References/MedicationRequest/requester.hbs ID=(generateUUID (toJsonString medReq.supply)) REF=(concat 'Organization/' orgId.Id)}}
                                     {{/with}}
                                 {{/if}}
-                            {{else if medEntry.substanceAdministration.author.assignedAuthor}}
+                            {{else if (and medReq.supply medEntry.substanceAdministration.author.assignedAuthor)}}
```

Comment:
added check that medReq.supply actually exists, since before was creating dangling references to a MedReq that was undefined